lib/perl/Genome/InstrumentData/AlignmentResult/Speedseq.t

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env genome-perl
+
+BEGIN {
+    $ENV{UR_DBI_NO_COMMIT} = 1;
+    $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+}
+
+use strict;
+use warnings;
+
+use above "Genome";
+use Test::More;
+use Genome::Test::Factory::InstrumentData::Solexa;
+use Genome::Test::Factory::Model::ImportedReferenceSequence;
+use Genome::Test::Factory::Build;
+use Genome::Test::Factory::SoftwareResult::User;
+use Sub::Override;
+
+my $pkg = 'Genome::InstrumentData::AlignmentResult::Speedseq';
+use_ok($pkg);
+
+my $test_data_dir = __FILE__.'.d';
+
+my $ref_seq_model = Genome::Test::Factory::Model::ImportedReferenceSequence->setup_object;
+my $ref_seq_build = Genome::Test::Factory::Build->setup_object(model_id => $ref_seq_model->id);
+use Genome::Model::Build::ReferenceSequence;
+my $override = Sub::Override->new(
+    'Genome::Model::Build::ReferenceSequence::full_consensus_path',
+    sub { return File::Spec->join($test_data_dir, 'human_g1k_v37_20_42220611-42542245.fasta'); }
+);
+use Genome::InstrumentData::AlignmentResult;
+my $override2 = Sub::Override->new(
+    'Genome::InstrumentData::AlignmentResult::_prepare_reference_sequences',
+    sub { return 1; }
+);
+
+my $instrument_data = Genome::Test::Factory::InstrumentData::Solexa->setup_object(
+    flow_cell_id => '12345ABXX',
+    lane => '2',
+    subset_name => '2',
+    run_name => 'example',
+    id => 'NA12878',
+);
+$instrument_data->bam_path(File::Spec->join($test_data_dir, 'NA12878.20slice.30X.bam'));
+
+my $result_users = Genome::Test::Factory::SoftwareResult::User->setup_user_hash(
+    reference_sequence_build => $ref_seq_build,
+);
+
+my $merged_alignment_result = Genome::InstrumentData::AlignmentResult::Merged::Speedseq->__define__();
+$merged_alignment_result->add_input(
+    name => 'instrument_data-1',
+    value_id => $instrument_data->id,
+);
+$merged_alignment_result->add_param(
+    name => 'instrument_data_count',
+    value_id=> 1,
+);
+$merged_alignment_result->add_param(
+    name => 'instrument_data_md5',
+    value_id => Genome::Sys->md5sum_data($instrument_data->id)
+);
+$merged_alignment_result->lookup_hash($merged_alignment_result->calculate_lookup_hash());
+my $override4 = Sub::Override->new(
+    'Genome::InstrumentData::AlignmentResult::Merged::Speedseq::merged_alignment_bam_path',
+    sub { return File::Spec->join($test_data_dir, 'merged_alignment_result.bam'); }
+);
+
+my $alignment_result = $pkg->create(
+    instrument_data => $instrument_data,
+    reference_build => $ref_seq_build,
+    picard_version => '1.46',
+    samtools_version => 'r963',
+    aligner_name => 'speedseq',
+    aligner_version => 'test',
+    aligner_params => {},
```

Comment:
`aligner_params` is a `Text` property--I'm surprised it accepts an empty hash reference.